lang/java/code_keyword.talon-list

```diff
@@ -0,0 +1,64 @@
+list: user.code_keyword
+code.language: java
+-
+
+abstract: "abstract "
+assert: "assert "
+boolean: "boolean "
+break
+byte: "byte "
+case: "case "
+catch
+char: "char "
+class: "class "
+continue
+default
+do: "do "
+double: "double "
+else
+enum: "enum "
+exports: "exports "
+extends: " extends "
+final: "final "
+finally: "finally "
+float: "float "
+for: "for "
+if: "if "
+implements: " implements "
+import: "import "
+instanceof: "instanceof "
+int: "int "
+interface: "interface "
+long: "long "
+module: "module "
+native: "native "
+new: "new "
+open: "open "
+opens: "opens "
+package: "package "
+private: "private "
+protected: "protected "
+provides: "provides "
+public: "public "
+requires: "requires "
+return: "return "
+short: "short "
+static: "static "
+strictfp: "strictfp "
```

Comment:
I am pretty sure Conformer is not going to work with this.

Reply:
If you want me to change it, could you make a suggestion on what to change it to?